docs/howtouse/sitename.rst

```diff
@@ -1,35 +1,22 @@
-Adding Sites
-============
+Updating the Domain of Your Site
+================================
 
 If you are running the record transfer app somewhere other than your localhost, you will need to add
 your site's domain to the database. If you do not set the domain, your emails will not send
 correctly, and will have the wrong link to your site.
 
-To set the name and domain of your site, first log in as a **superuser**. Regular staff users do not
-have permission to modify Sites.
+To set the domain of your site, run the following command from the `app` directory:
```

Comment:
It should be made explicit that this needs to be run from the `app` directory inside the running container. Someone might confuse this for needing to run it locally.